tests/test_thermostat.py

```diff
@@ -126,7 +126,7 @@ def test_get_data(self):
         response_data = load_fixture("thermostat.json")
         responses.add(
             responses.GET,
-            config.THERMOSTAT_URL,
+            util.get_thermostat_url(),
```

Comment:
with the suggested change, the mocks would have to be moved below the initialization of `thermostat`, and then could just be...

```suggestion
            thermostat._url,
```